workitems/l2-transaction-fees/l2-transaction-fees-v1.0-psd01.md

```diff
@@ -0,0 +1,334 @@
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[1.3.1	Requirement Ids](#131-requirement-ids) \
+[2 Concepts and Design](#2-Concepts-and-Design) \
+[3 Layer 2 Transaction Fees Specification](#3-l2-transaction-fees-specification) \
+[4 Conformance](#4-conformance) \
+&nbsp;&nbsp;&nbsp;&nbsp;[4.1 Conformance Targets](#41-conformance-targets) \
+&nbsp;&nbsp;&nbsp;&nbsp;[4.2 Conformance Levels](#42-conformance-levels)\
+[Appendix A - References](#appendix-a---references)\
+&nbsp;&nbsp;&nbsp;&nbsp;[A.1 Normative References](#a1-normative-references) \
+&nbsp;&nbsp;&nbsp;&nbsp;[A.2 Non-Normative References](#a2-non-normative-references) \
+[Appendix B - Security Considerations](#appendix-b---security-considerations) \
+&nbsp;&nbsp;&nbsp;&nbsp;[B.1 Data Privacy](#b1-data-privacy) \
+&nbsp;&nbsp;&nbsp;&nbsp;[B.2 Production Readiness](#b2-production-readiness) \
+[Appendix C - Acknowledgments](#appendix-c---acknowledgments)\
+[Appendix D - Revision History](#appendix-d---revision-history)\
+[Appendix E - Notices](#appendix-e---notices)
+
+ 
+
+-------
+# 1 Introduction
+
+The L2 WG is an open-source initiative with a scope to
+- Identify and document the most relevant use cases and business requirements for Layer 2 and other Blockchain Scalability solutions for EVM compatible public blockchains
+- Define a technical standard with identification and differentiation of classes of scalability solutions as required that meet both ecosystem and enterprise requirements, with a particular focus on interoperability between Layer 2 solutions for EVM compatible public blockchains
+- For EVM compatible public blockchains, identify, document, and devise solution approaches for Layer 2 Blockchain scalability solution specific challenges such as MEV, block (gas) limits, TVL concentration, etc.
+- Identify and document characteristics of Layer 2 Blockchain environments for EVM compatible public blockchains that will be key in addressing mainstream and enterprise adoption.
+
+The work is an [EEA Community Project](https://entethalliance.org/eeacommunityprojects/), which is managed by [OASIS](https://oasis-open-projects.org/).
+
+## 1.1 Overview
+
+... 
+
+
+## 1.2 Glossary
+
+**Blockchain:**
+
+An open, distributed ledger that can record transactions between two parties efficiently and in a verifiable and permanent way.
+
+**Gas**
+
+Refers to the unit that measures the amount of computational and storage effort required to execute specific operations on an EVM-compatible network.
+
+**Base Fee**
+
+The Layer 2 gas price. 
+
+Note that a gas price is typically variable and changes with the level of usage of a Layer 2 network and is expressed in Giga Wei (GWEI) for EVM-compatible networks, where Wei represents the smallest unit of gas; 1 Wei = 10<sup>-18</sup> Eth in Ethereum's native token. Also note, that typically there is a minimal, fixed base fee which normally varies by Layer 2 network. 
+
+**Execution Fee**
```

Comment:
The *fee* should be the *gas price* times the *amount of gas used*. 

For example, see https://help.optimism.io/hc/en-us/articles/4411895794715-How-do-transaction-fees-on-Optimism-work-

Reply:
said `gas value` to be more precise. Also, the gas price is the base fee unless the gas price is less than the minimum base fee. There might be variances in this definition if there are for example sequencer fees for fraud-proof challenges or aggregator fees in the L2 that are in addition to the gas used in the EVM. Hence, the vague statement.